### PR TITLE
added multiple css handles

### PR DESCRIPTION
**Pull request by claudiu-iviteb**
#### What problem is this solving?

It gives you the possibility to better style the components.

#### How to test it?

Link the app.

[Workspace](https://prreviews--iviteb.myvtex.com/papusa-3/p)

#### Screenshots or example usage:

image

#### Describe alternatives you've considered, if any.

Tryed styling it using current classes.

### CODE REVIEW

File: react/ReviewForm.tsx [hunk 1]
```diff
@@ -461,14 +472,14 @@ export function ReviewForm({ settings }: { settings?: Partial<AppSettings> }) {
...
-            <div className="mv3">
+            <div className={`${handles.formSection} ${handles.formSubmit} mv3`}>
               <Fragment>
                 {state.showValidationErrors &&
                   (!state.validation.hasName ||
                     !state.validation.hasTitle ||
                     !state.validation.hasText ||
                     !state.validation.hasValidEmail) && (
-                    <div className="c-danger t-small mt3 lh-title">
+                    <div className={`${handles.formInvalidMessage} "c-danger t-small mt3 lh-title"`}>
```
Comment (kevinch) [inline react/ReviewForm.tsx:482]:
```suggestion
                    <div className={`${handles.formInvalidMessage} c-danger t-small mt3 lh-title`}>
```

File: react/Reviews.tsx [hunk 2]
```diff
@@ -629,29 +640,35 @@ function Reviews() {
...
                           </strong>
                           {state.settings &&
                             state.settings.useLocation &&
                             review.location && <span>, {review.location}</span>}
-                        </li>
-                      </ul>
-                      <div className="t-body lh-copy mw9">
+                        </span>
+                      </div>
+                      <div className={`${handles.reviewCommentMessage}"t-body lh-copy mw9"`}>
```
Comment (kevinch) [inline react/Reviews.tsx:671]:
```suggestion
                      <div className={`${handles.reviewCommentMessage} t-body lh-copy mw9`}>
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,13 +5,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+### Added
+
+- `formSection`, `formBottomLine`, `formRating`, `formName`, `formLocation`, `formEmail`, `formReview`, `formSubmit`, `formInvalidMessage`, `reviewCommentMessage`, `reviewsOrderBy`, `reviewInfo`, `reviewVerifiedPurchase`, `reviewDate`, `reviewDateSubmitted`, `reviewDateValue`, `reviewAuthor`, `reviewAuthorBy`, `reviewAuthorName`, `summaryTotalReviews` and `writeReviewButton` CSS handles.
+### Changed
 
 ## [2.10.2] - 2021-06-02
 
 ### Fixed
 
 - Fix for anonymous review submission
```
Comment (kevinch) [inline CHANGELOG.md:18]:
```suggestion
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -5,13 +5,18 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
 
+- Review details structure (author and date) from using ul to div and spans.
```
Comment (kevinch) [inline CHANGELOG.md:19]:
```suggestion
- Review details structure (author and date) from using `ul` to `div` and `span`s.

```

Comment (kevinch) [reply CHANGELOG.md:19]: Looks like prettier is missing from this readme file.

**Review by polishq — APPROVED** [review]

**Review by igorbrasileiro — APPROVED** [review]
lgtm

File: CHANGELOG.md [hunk 5]
```diff
@@ -5,12 +5,17 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+
+- `formSection`, `formBottomLine`, `formRating`, `formName`, `formLocation`, `formEmail`, `formReview`, `formSubmit`, `formInvalidMessage`, `reviewCommentMessage`, `reviewsOrderBy`, `reviewInfo`, `reviewVerifiedPurchase`, `reviewDate`, `reviewDateSubmitted`, `reviewDateValue`, `reviewAuthor`, `reviewAuthorBy`, `reviewAuthorName`, `summaryTotalReviews` and `writeReviewButton` CSS handles.
+### Changed
 
 ## [2.10.2] - 2021-06-02
 
 ### Fixed
 
 - Fix for anonymous review submission
+- Review details structure (author and date) from using `ul` to `div` and `span`s.
```
[the remark this answers is not in the document]
Comment (claudiu-iviteb) [reply CHANGELOG.md]: Sorry, my mistake. Fixed

File: react/RatingSummary.tsx [hunk 6]
```diff
@@ -227,7 +228,7 @@ function RatingSummary() {
             <Stars rating={state.average} />
           </span>{' '}
           {state.settings.displaySummaryTotalReviews ? (
-            <span className="review__rating--count dib v-mid">
+            <span className={`${handles.summaryTotalReviews} review__rating--count dib v-mid`}>
```
[the remark this answers is not in the document]
Comment (claudiu-iviteb) [reply react/RatingSummary.tsx:231]: What issue? Can you be a little more specific? Thank you!

[the remark this answers is not in the document]
Comment (claudiu-iviteb) [reply react/RatingSummary.tsx]: Done

**Review by Mariana-Caetano — COMMENTED** [review]
Great work, @claudiu-iviteb! 
At end of the first paragraph on `readme.md` it has points of ellipsis and a better way to finish the sentence would be a period in the end. I would suggest a change for the following:

> Reviews & Ratings is a VTEX IO native solution that allows shoppers to submit reviews and ratings for products, as well as see them while navigating the store.

**Review by Mariana-Caetano — APPROVED** [review]
LGTM

**Pull request merged**